core/templates/dev/head/pages/Base.js

```diff
@@ -21,8 +21,10 @@ oppia.controller('Base', [
   'SidebarStatusService',
   function($scope, $rootScope, $document, AlertsService, BackgroundMaskService,
       SidebarStatusService) {
+    $scope.additionalAngularModule = GLOBALS.ADDITIONAL_ANGULAR_MODULES;
```

Comment:
Rename: additionalAngularModule**s**

Reply:
is this an unused variable?

Reply:
I intended to leave it there for future usage. This variable really only comes to play once editor.py and reader.py are involved

Reply:
Renamed as suggested!